app/assets/javascripts/listings/components/browse-listings.html.slim

```diff
@@ -9,17 +9,17 @@ header.lead-header ng-if="$ctrl.parent.hasEligibilityFilters()"
         | {{ $ctrl.headerText() }}
       .results-header
         p.results-header_values
-          | For
+          | {{'t.for' | translate}}
           strong
             '  {{$ctrl.eligibilityFilters.household_size}}
           ' {{$ctrl.eligibilityFilters.household_size == 1 ? 'person ' : 'people '}}
           span ng-if="$ctrl.eligibilityFilters.children_under_6 && $ctrl.eligibilityFilters.children_under_6 > 0"
-            ' (including
+            ' ({{'t.including' | translate}}
```

Comment:
This part might be a bit complicated because the order of words and phrases don't always match up in different languages. I'll take a look at this bit in particular and ask for advice from a chines-language speaker